apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx

```diff
@@ -125,6 +125,8 @@ const ManagerDashboard = ({ userQuery }: ManagerDashboardProps) => {
                       setAccordionItems(newValue);
                     }}
                     type="multiple"
+                    // we don't need that fat padding in the accordion inside the task card
+                    data-h2-padding-bottom="base:selectors[>.Accordion__Item > .Accordion__Content](0)"
```

Comment:
I think you want this to be x.5 so it matches the space at the top of the box.
![image](https://github.com/user-attachments/assets/a2553276-05d1-4dbc-9caf-0338a393ff93)


```suggestion
                    data-h2-padding-bottom="base:selectors[>.Accordion__Item > .Accordion__Content](x.5)"
```

Another option could be to update the TaskCard.Item bottom padding that wraps everything, but this works as well.